engines/bops_api/app/services/bops_api/application/creation_service.rb

```diff
@@ -136,6 +136,45 @@ def process_ownership_certificate_details(planning_application)
           end
         end
       end
+
+      def possibly_immune?(planning_application)
+        # TODO make planning_application an instance variable to simply this method et seq.
+        planning_application.immune_proposal_details.count > 1
```

Comment:
I'd use `many?` as the `count` made me think that it was doing an SQL on an association but that's not the case as it's a `select` over an array returned from a jsonb field.